components/landing/LandingPage.vue

```diff
@@ -52,6 +53,7 @@
 
 <script lang="ts" setup>
 import type { Prefix } from '@kodadot1/static'
+import MobileLanding from '~/components/landing/MobileLanding.vue'
```

Comment:
auto import `LandingMobileLanding` consider renaming this component

Reply:
renamed